### PR TITLE
Popups config for gratispaideiawiki

### PR DESCRIPTION
**Pull request by ugochimobi**

### CODE REVIEW

**Comment by Reception123** [comment]
You forgot to add the default for all the settings you are adding.

File: LocalSettings.php [hunk 1]
```diff
@@ -3311,6 +3311,18 @@
 	'wgPopupsOptInDefaultState' => [
 		'default' => 0,
 	],
+	'wgPopupsConflictingNavPopupsGadgetName' => [
+                'default' => '',
```
Comment (Reception123) [inline LocalSettings.php:3315]: https://github.com/wikimedia/mediawiki-extensions-Popups/blob/master/extension.json#L67

File: LocalSettings.php [hunk 2]
```diff
@@ -3311,6 +3311,18 @@
...
+		'gratispaideiawiki' => 'Navigation_popups',
+	],
+	'wgPopupsConflictingRefTooltipsGadgetName' => [
+                'default' => '',
```
Comment (Reception123) [inline LocalSettings.php:3319]: https://github.com/wikimedia/mediawiki-extensions-Popups/blob/master/extension.json#L71

File: LocalSettings.php [hunk 3]
```diff
@@ -3311,6 +3311,18 @@
...
+		'gratispaideiawiki' => 'ReferenceTooltips',
+	],
+	'wgPopupsOptInStateForNewAccounts' => [
+                'default' => null,
```
Comment (Reception123) [inline LocalSettings.php:3323]: Already the default setting; no need for this (https://github.com/wikimedia/mediawiki-extensions-Popups/blob/master/extension.json#L63)

**Review by Reception123 — CHANGES_REQUESTED** [review]

**Comment by ugochimobi** [comment]
@Reception123 This should be done now

**Comment by Reception123** [comment]
@ugochimobi Sorry you keep getting confused about this. Let me try to explain again.

The default value is the one that exists in the extensions extension.json, and unless it is changed that is what it will be for all Miraheze wikis. If a wiki wants to have a different value than the one that is default that will have to be added, but the default value will also have to be.

So you need to do both the default (as it's shown in extension.json) AND the value you need for your wiki.

**Comment by Reception123** [comment]
no, just like you did https://github.com/miraheze/mw-config/commit/ab883efb06146b2a9d1c28fa187f1acf6e4cec8e

**Comment by ugochimobi** [comment]
okay, but if want I want is also the default, then I should add just default right?

**Comment by Reception123** [comment]
> okay, but if want I want is also the default, then I should add just default right?

No, because the default already applies because of extension.json, it doesn't need to be added separately to LS

**Comment by ugochimobi** [comment]
Ah, I don't know why I am doubting that, because there are so many things that are supposed to be happening (apart from this PR) which aren't and it's causing me to doubt

File: LocalSettings.php [hunk 4]
```diff
@@ -3311,6 +3311,15 @@
...
 		'default' => 0,
 	],
+	'wgPopupsConflictingNavPopupsGadgetName' => [
+                'default' => 'Navigation_popups',
+	],
+	'wgPopupsConflictingRefTooltipsGadgetName' => [
+                'default' => 'ReferenceTooltips',
+	],
+	'wgPopupsOptInStateForNewAccounts' => [
+                'default' => 1,
```
Comment (Universal-Omega) [inline LocalSettings.php:3321]: This should be a string.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]
+this is pointless, we are adding configs, setting only defaults that would already be done by the extension.
